src/NzbDrone.Core.Test/ParserTests/ReleaseGroupParserFixture.cs

```diff
@@ -47,11 +47,12 @@ public class ReleaseGroupParserFixture : CoreTest
         [TestCase("SomeMovie.1080p.BluRay.DTS.x264.-VH-PROD.mkv", "VH-PROD")]
         [TestCase("Some.Dead.Movie.2006.1080p.BluRay.DTS.x264.D-Z0N3", "D-Z0N3")]
         [TestCase("Movie.Title.2010.720p.BluRay.x264.-[YTS.LT]", "YTS.LT")]
+        [TestCase("The.Movie.Title.2013.720p.BluRay.x264-ROUGH [PublicHD]", "ROUGH")]
 
         //[TestCase("", "")]
 
         //Exception Cases
```

Comment:
Move the comment with the moved function, or remove it i guess? none of the others are commented and the names are pretty clear lol

Reply:
Cleaned it up